tests/data/geotab_msgs_2.json

```diff
@@ -0,0 +1,62 @@
+[
```

Comment:
The xxx_msgs_1 is referenced but not this _2 named file, do we need?

Reply:
This was supposed to be used, adding it back now